python/lib/db/query/notification.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import select
+from sqlalchemy.orm import Session as Database
+from lib.db.model.notification_type import DbNotificationType
+
+
+def get_notification_type_with_name(db: Database, name: str):
+    """
+    Get a notification type from the database using its configuration setting name, or raise an
+    exception if no notification type is found.
```

Comment:
do we want to raise an exception if notification type is not existant or do we want to create the notification type?

Reply:
The current code does not create the notification type, but I thought about it and I agree creating it would probably be the better option.

Reply:
yes, that is what I assumed. I think for now it is fine but we could probably add that in the future. The same way we create the parameter_type when it does not exist. I don't think this is high priority though.